index.js

```diff
@@ -0,0 +1,34 @@
+var express = require("express");
+var hbs = require("express-handlebars");
+var db = require("./db/connection");
+
+var app = express();
+
+app.set("view engine", "hbs");
+app.engine(".hbs", hbs({
+  extname: ".hbs",
+  partialsDir: "views/",
+  layoutsDir: "views/",
+  defaultLayout: "layout-main"
+}));
+app.use("/assets", express.static("public"));
+
+app.get("/", function(req, res){
+  var randComp1 = db.compliments[Math.floor(Math.random() * db.compliments.length)];
```

Comment:
Could you think of a way to accomplish this with just one function to grab a random compliment?